dev/MRTCore/SignConfig.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<SignConfigXML>
+ <job platform="x64" certSubject="CN=Microsoft Corporation, O=Microsoft Corporation, L=Redmond, S=Washington, C=US" configuration="Release" dest="__INPATHROOT__\x64\signed" jobname="MRT Core binary sign (x64)" approvers="">
+   <file src="__INPATHROOT__\x64\mrm.dll" signType="AuthenticodeFormer" /> 
+   <file src="__INPATHROOT__\x64\Microsoft.ApplicationModel.Resources.dll"  signType="AuthenticodeFormer"  /> 
```

Comment:
><file src="__INPATHROOT__\x64\Microsoft.ApplicationModel.Resources.dll"  signType="AuthenticodeFormer"  />  [](start = 3, length = 107)

Is this no longer needed?

Reply:
Is the package in the nuget feed packageES signed?  (I assume it is, non-blocking, just curious).

Reply:
yes.

---
In reply to: [488953961](https://github.com/microsoft/ProjectReunion/pull/185#discussion_r488953961) [](ancestors = 488953961)